faker/providers/address/__init__.py

```diff
@@ -82,3 +82,40 @@ def country_code(self, representation=ALPHA_2):
             return self.random_element(self.alpha_3_country_codes)
         else:
             raise ValueError("`representation` must be one of `alpha-2` or `alpha-3`.")
+
+    def administrative_unit(self):
+        for name in ["province", "region", "state", "prefecture", "county"]:
```

Comment:
I don't like this 'guessing'. Feels t0o magical. I'd rather force the providers to have an `administrative_unit` method by using unit tests.

Reply:
So just to be clear: you prefer me to go to each locale, find the attribute called "state", "province", etc.

Rename that to "administrative_unit"

Then alias it back to the original name?

Reply:
Correct. I do realize how labor-intensive (and boring) it is, but I think it would be more correct for `administrative_unit` to be "canonical" method, and localized terminology to just be a translation of the canonical name.